package _Exception;


public class Example02 {
	public static void main(String[] args) {
		try {
		//	Connection con = new Connection();
		} catch (Exception e) {
			
			
		}finally {
		//	con.close(); //her turlu calistigi için connection'ý finallyde kaparýz.
		//catch blogu calissa da calýsmasa da finally her turlu calýsýr. 
		}
		
	}

}
